src/specimens/Type.js

```diff
@@ -138,8 +143,8 @@ Type.propTypes = {
   weight: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   tracking: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   image: PropTypes.string,
-  headings: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
-  paragraphs: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+  headings: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.shape])),
+  paragraphs: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.shape])),
```

Comment:
I actually meant _describing_ the shape 🙃  (I think just using `PropTypes.shape` will throw)

See https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

Reply:
😔 I'm new to PropTypes... In my head I said "I guess 'shape' must be some abstract term for an object.." It's not, haha. I changed it to `PropTypes.object`

Reply:
Wait, now that I'm reading the docs more, do you actually want me to break down the object here? Ohhh